src/main/java/spoon/reflect/cu/CompilationUnit.java

```diff
@@ -128,6 +33,7 @@
 	 * 		an arbitrary index in the source code
 	 * @return the index where the line starts
 	 */
+	@Deprecated
```

Comment:
Could you specify on the deprecated method what should be used instead with a deprecated annotation in the javadoc.

Reply:
there is no better alternative for that unused (useless?) method. But may be it is safer to keep it there - without deprecated. WDYT?